moduleroot/.github/workflows/auto_release.yml.erb

```
@@ -35,83 +35,71 @@ jobs:
-
     - name: Append PDK to Gemfile
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       run: |
         echo "gem 'pdk'" >> Gemfile
 
-    - name: Bundler Setup
+    - name: Activate Ruby 2.7
+      uses: ruby/setup-ruby@v1
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
+      with:
+        ruby-version: "2.7"
+        bundler-cache: true
+
+    - name: Print bundle environment
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       run: |
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle config path vendor/gems' -- bundle config path vendor/gems
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle config bin vendor/bins' -- bundle config bin vendor/bins
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle config jobs 8' -- bundle config jobs 8
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle config retry 3' -- bundle config retry 3
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle install' -- bundle install
-        buildevents cmd $TRACE_ID $STEP_ID 'bundle clean' -- bundle clean
         echo ::group::bundler environment
         buildevents cmd $TRACE_ID $STEP_ID 'bundle env' -- bundle env
         echo ::endgroup::
 
     - name: "Honeycomb: Record Bundler Setup time"
-      if: ${{ always() }}
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       run: |
         buildevents step $TRACE_ID $STEP_ID $STEP_START 'Bundler Setup'
-        echo STEP_ID="auto_release" >> $GITHUB_ENV
+        echo STEP_ID="auto_release_setup" >> $GITHUB_ENV
         echo STEP_START=$(date +%s) >> $GITHUB_ENV
 
     - name: "PDK Release prep"
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       run: |
-        buildevents cmd $TRACE_ID $STEP_ID 'pdk release prep' -- ./vendor/bins/pdk release prep --force 
+        buildevents cmd $TRACE_ID $STEP_ID 'pdk release prep' -- bundle exec pdk release prep --force 
 
     - name: "Commit changes"
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       run: |
         git config --local user.email "action@github.com"
         git config --local user.name "GitHub Action"
         git add .
         git commit -m "Auto-Release prep"
 
     - name: "Get Version"
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       id: gv
       run: |
         echo "::set-output name=ver::$(cat metadata.json | jq .version)"
 
     - name: Create Pull Request
       id: cpr
       uses: puppetlabs/peter-evans-create-pull-request@v3
+      if: ${{ github.repository_owner == '<%= common['owner'] %>' }}
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        commit-message: "Auto-release prep"
+        commit-message: "Release prep v${{ steps.gv.outputs.ver }}"
         branch: "release-prep"
         delete-branch: true
-        title: "Auto-release prep ${{ steps.gv.outputs.ver }}"
+        title: "Release prep v${{ steps.gv.outputs.ver }}"
```

```suggestion
        title: "Release prep v${{ steps.gv.outputs.ver }}"
        body: "Automated release-prep through [pdk-templates](https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.github/workflows/auto_release.yml.erb)"
```

Avoid advertising for others when we can advertise for our own stuff.

---

extra points for extracting the generated changelog bits of this release and add them to the text here.